tox.ini

```diff
@@ -6,7 +6,8 @@ whitelist_externals = env
 install_command = pip install -U {opts} {packages}
 setenv= TOX_ENV_NAME={envname}
 passenv = TOX_*
-deps = -r{toxinidir}/requirements.txt
+deps = {py,py26,py27,py33,py34,py35,pypy}: pip
```

Comment:
this won't work because the error happens when pip is trying to install the deps, I added the pip installation to just upgrade pip on python versions where the error doesn't happen